tools/boilerplate.py

```diff
@@ -316,12 +318,13 @@ def format_value(value):
 
             # A gensym-like facility in case some function takes an
             # argument named washold, ax, or ret
-            washold, ret, ax = 'washold', 'ret', 'ax'
+            washold, ret = 'washold', 'ret'
```

Comment:
the comment above needs to be updated
side discussion: I think we should just throw an exception in this case -- the parameter name is part of the API so renaming it seems silly